src/python/phenix_apps/apps/scorch/kafka/README.md

```diff
@@ -0,0 +1,44 @@
+# minimega (kafka) Component
+Collects and filters desired Kafka data into CSV/ JSON format, and converts values to be eaisly importable to Excel. The output directory for the data is listed in the phenix log when the scorch component starts. 
+
+```
+type: kafka
+exe:  phenix-scorch-component-kafka
+```
+## Metadata Options
+
+```yaml
+metadata:
+    bootstrapServers: [<string>]
+    allTags: <bool> #boolean indicating if all tags should be scanned or if specific tags should be subscribed to
+    subscribeTags: <list> #list of strings that indiate which tags we will subscribe to
+    mode: <string>  #string that can be 'substation', 'critical load', or 'all data'
```

Comment:
Delete this - make "all" the default and check for presence of "data" key: this should be a regex value that searches the contents of the topic message

Reply:
("data" should replace 'critLoad' and 'substation')